monkey/tests/unit_tests/infection_monkey/network/relay/test_relay_connection_handler.py

```diff
@@ -0,0 +1,64 @@
+import socket
+from ipaddress import IPv4Address
+from unittest.mock import MagicMock
+
+import pytest
+
+from monkey.infection_monkey.network.relay import (
+    RelayConnectionHandler,
+    RelayUserHandler,
+    TCPPipeSpawner,
+)
+from monkey.infection_monkey.network.relay.relay_connection_handler import RELAY_CONTROL_MESSAGE
```

Comment:
You can export `RELAY_CONTROL_MESSAGE` from `relay/__init__.py` and simplify this import.